packages/dash-shaka-playback/index.html

```
@@ -0,0 +1,55 @@
+﻿<!DOCTYPE HTML>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)